doc/verification.rst

```diff
@@ -80,10 +80,17 @@ Getting Started
 Prerequisites & Environment Setup
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- VCS RTL simulator (needed to support UVM 1.2)
-- RISCV-DV Prerequisites - https://github.com/google/riscv-dv#prerequisites
-- GCC setup - https://github.com/google/riscv-dv#compile-generated-programs-with-gcc
-- ISS setup - https://github.com/google/riscv-dv#run-iss-instruction-set-simulator-simulation - note that commit log must be enabled in spike by passing ``--enable-commitlog`` to the configure script.
+In order to run the co-simulation flow, you'll need an RTL simulator,
+an instruction set simulator and a working RISCV compiler.
```

Comment:
Could add a link to the lowrisc toolchains that are available here: https://github.com/lowRISC/lowrisc-toolchains/releases?